.github/workflows/ci.yml

```diff
@@ -1,5 +1,36 @@
 name: CI
 
+env:
+  TESTING_PRIVATE_KEY: |
```

Comment:
Github sent a security warning about possibly publishing a secret here. It's nice that they have scans for this. I responded with the "it's a test value" option.